### PR TITLE
Experimentation

### PR DESCRIPTION
**Pull request by chicharr**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--<repo>--<owner>.hlx.page/
- After: https://<branch>--<repo>--<owner>.hlx.page/

### CODE REVIEW

File: fstab.yaml [hunk 1]
```diff
@@ -1,3 +1,2 @@
 mountpoints:
-  /: https://drive.google.com/drive/folders/0AIerlgs8oIQXUk9PVA
-
+  /: https://drive.google.com/drive/folders/1G1UJWpjbA-Iu2mRtH-Mz1JpAt9DA4GLz
```
Comment (ramboz) [inline fstab.yaml]: Let's make sure to remove that so we don't change their fstab by mistake

File: scripts/blog.js [hunk 2]
```diff
@@ -92,4 +92,4 @@ export default async function decorateTemplate(main) {
       ]),
     );
   }
-}
+}
```
Comment (ramboz) [inline scripts/blog.js:95]: Change is probably not needed

File: scripts/experimentation-ued.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  decisionPolicy.decisionNodes.forEach(function (item) {
+      decisionNodesMap[item['id']] = item;
+  });
+  var items = traverseDecisionTree(decisionNodesMap, context, decisionPolicy.rootDecisionNodeId);
+  return {
+      items: items
+  };
+}
+
+export { evaluateDecisionPolicy };
```
Comment (ramboz) [inline scripts/experimentation-ued.js]: Maybe add the empty line at the end so linting doesn't complain

File: scripts/integrations-listing.js [hunk 4]
```diff
@@ -135,4 +135,4 @@ export default async function decorateTemplate(main) {
   sections.forEach((section, i) => section.classList.add(`${classes[i]}-container`));
   setupListingTabs(main);
   populateListingDetails(main);
-}
+}
```
Comment (ramboz) [inline scripts/integrations-listing.js:138]: Again, likely not needed

File: scripts/performance.js [hunk 5]
```diff
@@ -4,91 +4,91 @@
  */
 
 function stamp(message, time = new Date() - performance.timing.navigationStart, type = '') {
-    // eslint-disable-next-line no-console
```
Comment (ramboz) [inline scripts/performance.js:7]: Did you actually change anything in there?

File: scripts/set-object.js [hunk 6]
```diff
@@ -15,4 +15,4 @@ export function setObject(obj, path, value) {
         : {},
     obj)[path[path.length-1]] = value;
   return obj;
-}
\ No newline at end of file
+}
```
Comment (ramboz) [inline scripts/set-object.js:18]: Likely not needed

File: scripts/set-object.min.js [hunk 7]
```diff
@@ -1,2 +1,2 @@
 // eslint-disable-next-line
-export function setObject(t,e,r){return Object(t)!==t||(Array.isArray(e)||(e=e.toString().match(/[^.[\]]+/g)||[]),e.slice(0,-1).reduce(((t,r,c)=>Object(t[r])===t[r]?t[r]:t[r]=Math.abs(e[c+1])>>0==+e[c+1]?[]:{}),t)[e[e.length-1]]=r),t}
\ No newline at end of file
+export function setObject(t,e,r){return Object(t)!==t||(Array.isArray(e)||(e=e.toString().match(/[^.[\]]+/g)||[]),e.slice(0,-1).reduce(((t,r,c)=>Object(t[r])===t[r]?t[r]:t[r]=Math.abs(e[c+1])>>0==+e[c+1]?[]:{}),t)[e[e.length-1]]=r),t}
```
Comment (ramboz) [inline scripts/set-object.min.js:2]: same

File: styles/styles.css [hunk 8]
```diff
@@ -14,6 +14,16 @@
 @import 'style-vars.css';
 @import 'backgrounds.css';
 @import 'themes/base.css';
+@import 'templates/payroll-roi.css';
```
Comment (ramboz) [inline styles/styles.css:17]: same

File: tools/preview/experimentation-preview.css [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+  background-color: #666;
+}
+
+.hlx-variant > div {
+  flex: 1 1 auto;
+}
+
+.hlx-variant > div.hlx-button {
+  flex: 0 0 auto;
+}
```
Comment (ramboz) [inline tools/preview/experimentation-preview.css]: maybe add an empty line here

File: scripts/performance.js (hunk 5 above)
Comment (chicharr) [reply scripts/performance.js:7]: I think all these changes are coming from the main branch of bamboohr. I merged latest `main` into my PR

File: tools/preview/experimentation-preview.css (hunk 9 above)
Comment (chicharr) [reply tools/preview/experimentation-preview.css]: done

**Pull request closed without merging**